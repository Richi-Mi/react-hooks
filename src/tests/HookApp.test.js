import React from 'react';
import { shallow } from "enzyme"
import HookApp from "../HookApp"

describe(' * Pruebas en <HookApp />', () => {
    
    const wrapper = shallow( <HookApp />)
    
    test(' - Debe mostrar el componente correctamente', () => {
        expect( wrapper ).toMatchSnapshot();
    });

})
