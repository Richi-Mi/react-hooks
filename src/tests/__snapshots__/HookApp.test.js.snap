// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` * Pruebas en <HookApp />  - Debe mostrar el componente correctamente 1`] = `
<div>
  <h1>
     Hola Mundo Hook 
  </h1>
</div>
`;
